### ISSUE TITLE
Dialog Button Alignment

### ISSUE THREAD

**Issue by ninavizz** [freedomofpress/securedrop-client#1353]
All dialog buttons should be right aligned. Currently, they are left aligned, both in Deletion and Export flows.

**Comment by ninavizz** [comment]
After reviewing the production Client more carefully against my prototypes and mockup(s), I think the problem is that they were more closely to center-aligned in my wireframes—which were done with dialogs much narrower, than they open in the client. 

Regardless: Both Delete and Export should be unified, once I have a spec ready. Sorry!

**Comment by gonzalo-bulnes** [comment]
@ninavizz Is the following an accurate representation of what you mean?

```
 _______________________________________________
|___________________Dialog____________________x_|
|#######padding######                           |
|#Some information, left aligned.              #|
|#                                             #|
|#                    ___________   __________ #|
| right aligned  --> |_Secondary_| |_Primary__|#|
|_______________________________###padding######|             
```

**Comment by ninavizz** [comment]
@gonzalo-bulnes Yes—however after closer study of my Zep-lin specs against the client in production, and newer needs presented in _Export All_, Allie and I just discussed and I think center aligned is better. More on this, shortly!

**Comment by eloquence** [comment]
Closing, non-critical issue specific to the current codebase/UX, which is undergoing a rewrite.
